libcxx/test/std/localization/locale.categories/category.monetary/locale.money.get/locale.money.get.members/get_long_double_fr_FR.pass.cpp

```diff
@@ -33,9 +33,9 @@
 
 #ifdef _AIX
 // the AIX libc expects U202F as LC_MONETARY thousands_sep
-#  define THOUSANDS_SEP L"\u202F"
+#  define THOUSANDS_SEP_ L"\u202F"
```

Comment:
Why did you make this change? It seems unrelated and unnecessary? Unrated changes should be done in separate patches.